src/models/appointments.js

```diff
@@ -1,4 +1,5 @@
 const { Schema, model } = require('mongoose');
+const Joi = require('joi');

```

Comment:
I'm not sure it's a good point to create schemas in models, because models regard to the database.

Reply:
Ok, I will move it to router file